### PR TITLE
Removed redundant specification of version number

### PR DESCRIPTION
**Pull request by fonnesbeck**
The package version currently resides in 4 different places. This change eliminates all but one, making maintenance easier.

### CODE REVIEW

**Comment by ColCarroll** [comment]
I'm pretty sure this fails on `pip install` because it will import pymc3 before installing requirements (trying `pip install -e .` locally fails because numpy isn't yet installed.  Will probably pass on travis since we install dependencies separately.

 I have used the pattern from https://github.com/pypa/pip/blob/master/setup.py to only define the version in 1 place before (essentially, read the `__init__.py` file, and use a regex to parse out the version).

**Comment by fonnesbeck** [comment]
I tested it locally with `pip install -e .` into a clean virtualenv and had no errors.

**Comment by twiecki** [comment]
@fonnesbeck that's because you ran this from the pymc3 root dir which allows `import pymc3` to work. I think a better solution is to use `versioneer`. https://github.com/warner/python-versioneer

**Comment by fonnesbeck** [comment]
No, this installs fine when installed from elsewhere too.

    $ pip install -e Repos/pymc3 

I'd prefer not to invoke a third-party package if possible.

**Comment by fonnesbeck** [comment]
I think as long as the import does not try to access any compiled code, this works fine.

**Comment by ColCarroll** [comment]
That's super strange. Importing pymc3 should import numpy et al.

**Comment by fonnesbeck** [comment]
@ColCarroll actually, I'm sure you are right. My virualenv is probably falling back on a global site-packages. I will try your regex approach.

File: setup.py [hunk 1]
```diff
@@ -3,15 +3,14 @@
...
+import re
 
 DISTNAME = 'pymc3'
 DESCRIPTION = "Probabilistic Programming in Python: Bayesian Modeling and Probabilistic Machine Learning with Theano"
 AUTHOR = 'PyMC Developers'
 AUTHOR_EMAIL = 'pymc.devs@gmail.com'
 URL = "http://github.com/pymc-devs/pymc3"
-LICENSE = "Apache License, Version 2.0"
-VERSION = "3.4rc1"
+LICENSE = "Apachimport numpye License, Version 2.0"
```
Comment (ColCarroll) [inline setup.py]: looks like stray characters here

**Comment by fonnesbeck** [comment]
Are we okay with this?

**Comment by ColCarroll** [comment]
Yep!

**Pull request merged**